src/components/calculator.js

```diff
@@ -1,78 +1,124 @@
 import { Component } from 'react';
+import calculate from './logic/calculate';
 
 export default class Calculator extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = { total: 0, next: null, operation: null };
   }
 
+  buttonClicked = (event) => {
+    const btnName = event.target.innerText;
+    const answer = calculate(this.state, btnName);
+    this.setState(answer);
+  };
+
   render() {
+    const { total, operation, next } = this.state;
     return (
       <div className="first">
         <div className="calculator-wrap">
-          <div className="calculator-display">0</div>
+          <div className="calculator-display">
+            {total}
+            {operation}
+            {next}
+          </div>
           <div className="calculator-keys">
-            <button type="button" className="all-clear" value="all-clear">
+            <button
+              type="button"
+              onClick={this.buttonClicked}
+              className="all-clear"
+              value="all-clear"
+            >
               AC
             </button>
-            <button type="button" className="operator" value="+/-">
+            <button
+              type="button"
+              onClick={this.buttonClicked}
+              className="operator"
+              value="+/-"
+            >
               +/-
             </button>
-            <button type="button" className="operator" value="%">
+            <button
+              type="button"
+              onClick={this.buttonClicked}
+              className="operator"
+              value="%"
+            >
               %
             </button>
-            <button type="button" className="operator" value="/">
+            <button
+              type="button"
+              onClick={this.buttonClicked}
+              className="operator"
+              value="/"
+            >
               &divide;
             </button>
-
-            <button type="button" value="7">
+            <button type="button" onClick={this.buttonClicked} value="7">
               7
             </button>
-            <button type="button" value="8">
+            <button type="button" onClick={this.buttonClicked} value="8">
               8
             </button>
-            <button type="button" value="9">
+            <button type="button" onClick={this.buttonClicked} value="9">
               9
             </button>
-
-            <button type="button" className="operator" value="*">
+            <button
+              type="button"
+              onClick={this.buttonClicked}
+              className="operator"
+              value="*"
+            >
               &times;
```

Comment:
- [ ] Good job implementing the lpogic for the app, however when we try to use the multiplication from the calculator it is giving an error please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/212074265-5c0cb49e-8e05-40bd-a4d9-57f1680abae5.png)

I think it is because you are using &times: HTML entity, please try to use `'x'` here the letter x.

-[ ] [OPTIONAL] I would suggest please stick to the original design provided in the project requirement 
![image](https://user-images.githubusercontent.com/97521557/212075094-465d7d06-20f7-40a5-a2d7-6f547b5d0aa9.png)